pytest-embedded-idf/pytest_embedded_idf/dut.py

```diff
@@ -127,3 +129,101 @@ def close(self) -> None:
         if not self.skip_check_coredump:
             self._check_coredump()
         super().close()
+
+    @dataclass
+    class Case:
```

Comment:
how about we change the name to `UnittestMenuCase`? The name `Case` would be a bit overbroad and is a widely used term in testing frameworks.